src/components/Dropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+import { Person } from '../types/Person';
+import { DropdownItem } from './DropdownItem';
+
+interface Props {
+  people: Person[];
+  searchName: string,
+  setSearchName: (value: string) => void,
+  setSelectedPersonSlug: (value: string) => void,
+  findSelectedPerson: Person | null,
+}
+
+export const Dropdown: FC<Props> = React.memo(({
+  people,
+  searchName,
+  setSearchName,
+  setSelectedPersonSlug,
+  findSelectedPerson,
+}) => {
+  return (
+    <div className="dropdown is-active">
+      <div className="dropdown-trigger">
+        <input
+          type="text"
+          placeholder="Enter a part of the name"
+          className="input"
+          value={searchName}
+          onChange={(event) => setSearchName(event.target.value)}
+        />
+      </div>
+
+      {searchName && searchName !== findSelectedPerson?.name && (
```

Comment:
not really a good idea to calculate it right here. Better to have it as a separate variable